### ISSUE TITLE
Add CodeSignatureVerification for signed apps

### ISSUE THREAD

**Issue by homebysix** [autopkg/keeleysam-recipes#48]
Would be great for those of us who are writing recipes that use yours as a ParentRecipe if you were to add CodeSignatureVerification to your .download recipes, where applicable. I'd be happy to help, if you're open to it.

**Comment by aysiu** [comment]
Not really sure how to add this download recipe for FileZilla and test it, but the code signature verification should be:
```identifier "de.filezilla" and anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists */ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* exists */ and certificate leaf[subject.OU] = "5VPGKXL75N"```

**Comment by grahampugh** [comment]
Adding CodeSignatureVerifier to FileZilla seems to have broken the recipe.
```
Processing FileZilla.jss...
FileZillaURLProvider
FileZillaURLProvider: Found URL https://dl1.cdn.filezilla-project.org/FileZilla_3.34.0_macosx-x86.app.tar.bz2
URLDownloader
URLDownloader: Storing new Last-Modified header: Fri, 15 Jun 2018 14:57:20 GMT
URLDownloader: Storing new ETag header: "5b23d3d0-a457c2"
URLDownloader: Downloaded /Users/---/Library/AutoPkg/Cache/local.jss.FileZilla/downloads/FileZilla.tar.bz2
EndOfCheckPhase
CodeSignatureVerifier
Error processing path '/Users/--/Library/AutoPkg/Cache/local.jss.FileZilla/downloads/FileZilla.app' with glob. 
Failed.
```

**Comment by aysiu** [comment]
Tried to address it with this PR:
https://github.com/autopkg/keeleysam-recipes/pull/92

**Comment by aysiu** [comment]
Of course, that might mean the .munki recipe's unarchiver processor may be redundant:
https://github.com/autopkg/keeleysam-recipes/blob/master/FileZilla/FileZilla.munki.recipe#L37-L49
